main.py

```diff
@@ -0,0 +1,66 @@
+#GRA W WISIELCA
+import random
+used_letters = []
+user_word = []
+
+word_tupla = []
```

Comment:
word_list to lepsza nazwa, nie okreslasz w nazwie zmeinnej jej typu. Jka miałbyś zmienną "B" która byłaby intem pisałbyś b_int ?